src/types.d.ts

```diff
@@ -95,10 +95,87 @@ declare global {
     catalogQuerySearchFiltersEnabled?: boolean,
   };
 
+  type SubscriptionPlan = {
+    title: string,
+    uuid: string,
+    startDate: string,
+    expirationDate: string,
+    enterpriseCustomerUuid: string,
+    enterpriseCatalogUuid: string,
+    isActive: boolean,
+    isCurrent: boolean,
+    isRevocationCapEnabled: boolean,
+    daysUntilExpiration: number,
+    daysUntilExpirationIncludingRenewals: number,
+    isLockedForRenewalProcessing: boolean,
+    shouldAutoApplyLicenses: boolean,
+    created: string,
+    planType: string,
+  };
+
+  type LearnerProfileSubscriptionType = {
```

Comment:
nit: No need to put 'Type' in the type name, as types are generally pretty easy to spot from the context they're used in.